docs/index.rst

```diff
@@ -4,4 +4,315 @@ JDBC Connector
+   sqlite>
+
+Next in the SQLite command prompt, create a table and seed it with some data::
+
+   sqlite> CREATE TABLE accounts(id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, name VARCHAR(255) NOT NULL);
+   sqlite> INSERT INTO accounts(name) VALUES('alice');
+   sqlite> INSERT INTO accounts(name) VALUES('bob');
+
+Now, create a configuration file that will load data from this database in
+``quickstart-sqlite.properties``::
+
+   name=test-sqlite-jdbc-autoincrement
+   connector.class=io.confluent.connect.jdbc.JdbcSourceConnector
+   tasks.max=1
+   connection.url=jdbc:sqlite:test.db
+   mode=increasing
+   increasing.column.name=id
+   topic.prefix=test-sqlite-jdbc-
+
+The first few settings are common settings you'll specify for all connectors. The ``connection.url``
+specifies the database to connect to, in this case a local SQLite database file. The ``mode``
+indicates how we want to query the data. In this case, we have an auto-incrementing unique
+ID, so we select ``increasing`` mode where each query uses the largest ID previously seen to limit
+new results to newly created rows. The ``increasing.column.name`` setting specifies the column in
+the table that contains the auto-incrementing ID. Finally, we can control the names of the topics
+that each table's output is sent to with the ``topic.prefix`` setting. Since we only have one
+table, the only output topic in this example will be ``test-sqlite-jdbc-accounts``.
+
+Now, run the connector in a standalone Kafka Connect worker in another terminal (this assumes
+Avro settings and that Kafka and the Schema Registry are running locally on the default ports)::
+
+   $ ./bin/connect-standalone ./config/connect-standalone.properties quickstart-sqlite.properties
+
+You should see the process start up and log some messages, and then it will begin executing
+queries and sending the results to Kafka. In order to check that it has copied the data that was
+present when we started Kafka Connect, start a console consumer, reading from the beginning of
+the topic::
+
+   $ ./bin/kafka-avro-console-consumer --new-consumer --bootstrap-server localhost:9092 --topic test-sqlite-jdbc-accounts --from-beginning
+   {"id":1,"name":{"string":"alice"}}
+   {"id":2,"name":{"string":"bob"}}
+
+The output shows the two records as expected, in the JSON encoding of the Avro records. Now,
+keeping that running, add another record via the SQLite command prompt::
+
+   sqlite> INSERT INTO accounts(name) VALUES('cathy');
+
+If you switch back to the console consumer, you should see the new record added (and,
+importantly, the old entries are not repeated)::
+
+   {"id":3,"name":{"string":"cathy"}}
+
+Note that the default polling interval is 5 seconds, so it may take a few seconds to show up.
+Depending on your expected rate up updates or desired latency, a smaller poll interval could be
+used to deliver updates more quickly.
+
+Of course, all the normal features of Kafka Connect, including offset management and fault
```

Comment:
Link to KafkaConnect docs here so people will be able to check out the "normal features".

Also, s/normal/awesome/ :)